api-tests/src/steps/cri-steps.ts

```diff
@@ -54,7 +54,7 @@ const submitAndProcessCriAction = async (
 };
 
 When(
-  /^I submit (expired )?'([\w-]+)' details to the (async )?CRI stub$/,
+  /^I submit (expired )?'([<>\w-]+)' details to the (async )?CRI stub$/,
```

Comment:
Do we need to include `<>` in this capture group? Can't see them being used anywhere?

Reply:
oops forgot to remove this, thanks! This was originally here to match the steps in a `Scenario Outline` but have since refactored